### PR TITLE
Add quickstart tests

### PR DESCRIPTION
**Pull request by granluo**
Add quickstart testings.

### CODE REVIEW

File: .github/workflows/remoteconfig.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: remoteconfig
```
Comment (paulb777) [inline .github/workflows/remoteconfig.yml:1]: Looks like master wasn't updated before this branch was created.

One way to fix would be `git fetch` and `git rebase origin/master`

File: .github/workflows/dynamiclinks.yml [hunk 2]
```diff
@@ -36,3 +36,22 @@ jobs:
       run: scripts/setup_bundler.sh
     - name: PodLibLint Storage Cron
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseDynamicLinks.podspec --platforms=ios ${{ matrix.flags }}
+
+  quickstart:
```
Comment (paulb777) [inline .github/workflows/dynamiclinks.yml:40]: The test is failing. There may be missing steps in the setup. It should be sorted out or removed for a subsequent PR.

File: .github/workflows/remoteconfig.yml (hunk 1 above)
Comment (paulb777) [inline .github/workflows/remoteconfig.yml:1]: It looks like master wasn't updated before this PR.

One way to fix would be to `git fetch` and `git rebase origin/master` on add_quickstart_tests branch.

File: .github/workflows/dynamiclinks.yml (hunk 2 above)
Comment (granluo) [reply .github/workflows/dynamiclinks.yml]: remove it for now will add it back later.

File: .github/workflows/dynamiclinks.yml [hunk 3]
```diff
@@ -3,8 +3,8 @@ name: dynamiclinks
 on:
   pull_request:
```
Comment (paulb777) [inline .github/workflows/dynamiclinks.yml:4]: Undo changes in this file - `git checkout master -- .github/workflows/dynamiclinks.yml`

**Pull request merged**